qiime2/sdk/result.py

```diff
@@ -238,8 +238,13 @@ def clone_original(into):
             # directory is empty, this function is meant to fix that, so we
             # can rmdir so that copytree is happy
             into.rmdir()
-            shutil.copytree(str(self._archiver.data_dir), str(into),
-                            copy_function=os.link)  # Use hardlinks
+            try:
+                shutil.copytree(str(self._archiver.data_dir), str(into),
+                                copy_function=os.link)  # Use hardlinks
+            except shutil.Error:
+                # Try again with full copy not links
+                shutil.copytree(str(self._archiver.data_dir), str(into),
+                                dirs_exist_ok=True)
```

Comment:
```suggestion
                                copy_function=qiime2.util.duplicate)
```

I don't know for certain that this will work, but it seems like it ought to. (and might inherit future symlink behavior).